MagaluCloud/magalu
# chore: rename 'x-cli' extension to 'x-mgc'

## Description

This makes more sense, as the extensions aren't specific to the CLI, but to the SDK as a whole

## Related Issues

- Closes #466

### Pull request checklist


## How to test it

Check some endpoints that use the flag to see if it's working. Use `-w` when reading a newly-created virtual machine instance, see if the default output for `virtual-machine instance list` is a table, things like that
